app/backend/db/crud.py

```diff
@@ -6,9 +6,23 @@
 from db import schemas
 
 # User:
+def create_user(db: Session, username: str, password: str) -> User:
+    new_user = User(english_name=username,
+                    username=username,
+                    password=password)
+    db.add(new_user)
+    db.commit()
+    db.refresh(new_user)
```

Comment:
why is the refresh needed?

Reply:
to get the user's id

Reply:
when i checked it on the ipython, when you commit the change, it already puts the id inside your user object